### DISCUSSION TITLE
Miss the old README.MD

### DISCUSSION THREAD

**Discussion by mschabhuettl** [tteck/Proxmox#1084] [Ideas]
Hello!

I really miss the old README.MD where I was able to quickly spot new install scripts.
Would it be possible to add the list back to README.MD?
Another option would be a curated "WhatsNew.md" of some sort, where the last (idk 10 new) install scripts are listed.
Also an "Changelog.md" would be an option - and now that I'm thinking of this, it may be the best and coolest way.

Cheers

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/blob/main/CHANGELOG.MD

**Reply by mschabhuettl** [reply]
Well. I'm stupid. :-D
Thank you!
